nflow-rest-api/src/main/java/io/nflow/rest/v1/WorkflowInstanceResource.java

```diff
@@ -169,19 +177,40 @@ public ListWorkflowInstanceResponse fetchWorkflowInstance(
         .setIncludeCurrentStateVariables(includes.contains(currentStateVariables)) //
         .setIncludeActions(includes.contains(actions)) //
         .setIncludeActionStateVariables(includes.contains(actionStateVariables)) //
-        .setMaxResults(maxResults) //
+        .setMaxResults(maxResults) // // TODO: move to include parameters in next major version
+
         .setMaxActions(maxActions) //
         .setIncludeChildWorkflows(includes.contains(childWorkflows)).build();
     Collection<WorkflowInstance> instances = workflowInstances.listWorkflowInstances(q);
     List<ListWorkflowInstanceResponse> resp = new ArrayList<>();
+    Set<WorkflowInstanceInclude> parseIncludeSet = parseIncludeSet(include);
+    // TODO: move to include parameters in next major version
+    parseIncludeSet.add(WorkflowInstanceInclude.STARTED);
     for (WorkflowInstance instance : instances) {
-      resp.add(listWorkflowConverter.convert(instance, q));
+      resp.add(listWorkflowConverter.convert(instance, parseIncludeSet));
     }
     return resp;
   }
 
-  private List<String> parseIncludes(String include) {
-    return asList(trimToEmpty(include).split(","));
+  private Set<WorkflowInstanceInclude> parseIncludeSet(String include) {
+    return asList(trimToEmpty(include).split(",")).stream().map(i -> {
+      switch (i) {
+      case childWorkflows:
```

Comment:
Create a static final Map<String, WorkflowStateInclude> map and just do `.map(toIncludeMap::get)` ?